packages/SwingSet/src/types.js

```diff
@@ -3,3 +3,25 @@
  * @property {string} body
  * @property {Array<string>} slots
  */
+
+/**
+ * @typedef {{
+ *   bundle: Bundle,
+ *   enableSetup: false,
+ * }} HasBundle
+ * @typedef {{
+ *   setup: unknown,
+ *   enableSetup: true,
+ * }} HasSetup
+ *
+ * TODO: metered...
```

Comment:
Would be good to replace this TODO with an issue link, and link the issue back to this file.